pilot/pkg/model/sidecar_test.go

```diff
@@ -76,12 +76,24 @@ var (
 		Spec: &networking.Sidecar{
 			Egress: []*networking.IstioEgressListener{
 				{
-					Hosts: []string{"foo/bar", "*/*"}, // MBMBMB not valid; one no-port egress
+					Hosts: []string{"foo/bar", "*/*"},
 				},
 			},
 		},
 	}
 
+	configs4 = &Config{
+		ConfigMeta: ConfigMeta{
+			Name:      "foo",
+			Namespace: "not-default",
+		},
+		Spec: &networking.Sidecar{
+			Egress: []*networking.IstioEgressListener{
+				{},
```

Comment:
I think this is not a valid Sidecar, have a look at `ValidateSidecar`

Reply:
That is correct. Not sure why I had this added in my earlier patches that I submitted a couple of weeks ago. The only thing useful here is removing the extraneous comment. So I am keeping the PR as that comment ought to be removed.